eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -105,6 +105,38 @@ function Get-dotnet-OnboardedDocsMsPackages($DocRepoLocation) {
     return $onboardedPackages
 }
 
+# The function use docs package info as a key and metadata/package.json as value to build a map.
+function Get-dotnet-OnboardedDocsMsPackagesForMoniker ($DocRepoLocation, $moniker) {
+    $onboardingSpec = ""
+    if ("preview" -eq $moniker) {
+        $onboardingSpec = Get-Content "$DocRepoLocation/bundlepackages/azure-dotnet-preview.csv"
+    }
+    elseif("latest" -eq $moniker) {
+        $onboardingSpec = Get-Content "$DocRepoLocation/bundlepackages/azure-dotnet.csv"
+    }
+    $onboardedPackages = @{}
+    foreach ($spec in $onboardingSpec) {
+        $packageInfo = $spec -split ","
```

Comment:
There's a parser that'll give you back a list of package objects (including package names) given an onboarding CSV file. https://github.com/Azure/azure-sdk-for-net/blob/main/eng/scripts/Language-Settings.ps1#L310-L365 .. Can we use that one?